### DISCUSSION TITLE
Logic Level Shifting of Logic Analyzer

### DISCUSSION THREAD

**Discussion by Curt-is-Pi** [fhdm-dev/scoppy#157] [Q&A]
Is using a TI 74AHCT125 a suitable way to allow Scoppy to analyze 5 volt logic?

**Comment by fhdm-dev** [comment]
I don't think so. The output voltage will be between 4.5V and 5V (the recommended range of the supply voltage) and of course you want the maximum output voltage to be less than 3.3V.

I have tried the SN74CB3T3245 and it works really well. However, it's more expensive and is not available as a through hole part.

**Reply by Curt-is-Pi** [reply]
I was hopful that the 74ahct125 would tolerate being fed with 3.3 volts and be accurate because I have them lying around. 
Thank you for the reply and suggesting a suitable component.
